distribution/docs/src/main/jdocs/content/_quickstart/quickstart-certificates.adoc

```diff
@@ -0,0 +1,121 @@
+:title: Certificates Quickstart
+:type: quickStart
+:status: published
+:summary: Keystore and certificate instructions.
+:order: 01
+
+=== Certificates (Quick Start)
+
+${branding} comes with a default keystore that contains certificates.
+This allows the distribution to be unzipped and run immediately.
+If these certificates are sufficient for testing purposes, proceed to <<_configuring_quick_start,Configuring (Quick Start)>>.
+
+To test federation using 2-way TLS, the default keystore certificates will need to be replaced, using either the included <<_demo_certificate_authority_ca,Demo Certificate Authority>> or by <<_creating_self_signed_certificates,Creating Self-signed Certificates>>.
+
+If the installer was used to install the ${branding} and a hostname other than "localhost" was given, the user will be prompted to upload new trust/key stores.
+
+If the hostname is `localhost` or, if the hostname was changed _after_ installation, the default certificates will not allow access to the ${branding} instance from another machine over HTTPS (now the default for many services).
+The Demo Certificate Authority will need to be replaced with certificates that use the fully-qualified hostname of the server running the ${branding} instance.
+
```

Comment:
Rick and I came up with a flow chart for use cases. I reworked it for the PRODUCTION scenario. I'm attaching the flow chart as an SVG file. I have the graphml, which can be converted to ASCII art. I'm trying to make a (graphical) recipe that people can follow. The chart is not all that simple as I would like, but it covers several different ways to reach a correct configuration. Please comment on it.
[certificate-flowchart.pdf](https://github.com/codice/ddf/files/1137029/certificate-flowchart.pdf)